TWLight/users/helpers/authorizations.py

```diff
@@ -59,35 +86,33 @@ def sort_authorizations_into_resource_list(authorizations):
     if authorizations:
         for authorization in authorizations:
             for partner in authorization.partners.all():
-                # Name this item according to whether this authorization is
-                # to a stream
                 stream = authorization.stream
-                if stream:
-                    name_string = stream.name
+                partner_streams = Stream.objects.filter(partner=partner)
+                if partner_streams and not stream:
```

Comment:
If a user is authorized to access a stream, they should have an authorization with that stream in it.

Reply:
Oh, yep. Fixing this the 'proper' way would mean going through the whole `partner`->`partners` thing with streams too, right?

The alternative being we always split partners up into their respective streams if they're switching to Bundle?

Reply:
I think the latter would only need to be done for Oxford University Press - it's the only partner with streams that's moving to Bundle.

Reply:
In that case, the easiest thing to do would be to just make a new OUP bundle partner, and mark the current one as unavailable.

Reply:
Oh that's a good point, since we don't care about existing apps/auths. I think it's something like 10 collections, which means nearly half of the Library Bundle will be individual OUP-published sources, but this may be the best option.

Reply:
FYI, our current OUP ezproxy config:
https://github.com/WikipediaLibrary/twlight_ezproxy/blob/19dd11e4f6947abdd90699ffda536117a83a2aac/ezproxy.cfg#L257-L272

Reply:
If we wanted to split it out, each one of those oxford entries could be a partner.

Reply:
Or they could all be a single partner

Reply:
Will clarify the full list with Nikki,

Reply:
Nikki's going to move forward with creating new partners for OUP. I'll revert this change and we'll move ahead with not considering Bundle streams at all.

Reply:
Oh and she's going to clarify the list of collections so that we can update the proxy config.